redbot/cogs/economy/economy.py

```diff
@@ -284,9 +289,9 @@ def __init__(self, bot: Red):
                     ).format(
                         author=author,
                         currency=credits_name,
-                        amount=await self.config.PAYDAY_CREDITS(),
-                        new_balance=await bank.get_balance(author),
-                        pos=pos,
+                        amount=humanize_number(await self.config.PAYDAY_CREDITS()),
+                        new_balance=humanize_number(await bank.get_balance(author)),
+                        pos=humanize_number(pos) if pos else pos,
```

Comment:
When can pos be zero? If there is a legitimate case for this, please add a comment explaining it :)

Reply:
in `get_leaderboard_position` there is a `return None` so there is a chance of it being None rather than 0 , it seems to use `discord.utils.find` which can return None... Doubt it could ever happen from a quick glance at the code, but since it's in the source function just wanted to handle it.

Reply:
Ah no problem :+1: